packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -2975,11 +2975,16 @@ String describeIdentity(Object? object) => '${objectRuntimeType(object, '<optimi
 ///
 /// void validateDescribeEnum() {
 ///   assert(Day.monday.toString() == 'Day.monday');
+///   // ignore: deprecated_member_use
 ///   assert(describeEnum(Day.monday) == 'monday');
 ///   assert(Day.monday.name == 'monday'); // preferred for real enums
 /// }
 /// ```
 /// {@end-tool}
+@Deprecated(
```

Comment:
Can dart fix be added for this?

Reply:
I don't know. Can Dart fix replace `describeEnum(something)` with `something.name`?

Reply:
It might be, often the answer is to try it and see what happens. `replacedBy` might be the way to go, and if you look up the `nullOk` dart fix rules, that should be a good example of how to grab the argument.